client/src/components/Header/Header.js

```diff
@@ -11,7 +11,7 @@ import Nav from '../Nav';
 const Header = () => {
   return (
     <header>
-      <Nav />
+      {Auth.loggedIn() && <Nav />}
```

Comment:
I think we'll want to take my version instead--I leave 40px of space there so the title is still centered

Here's how it looks on this branch:
<img width="1439" alt="Screen Shot 2022-04-23 at 9 13 51 PM" src="https://user-images.githubusercontent.com/16024741/164956307-90b47c2f-02a4-4d64-b2dc-fa6309fe9a3c.png">
.